crates/ra_analysis/src/lib.rs

```diff
@@ -120,7 +121,7 @@ pub struct SourceChange {
 #[derive(Debug)]
 pub struct SourceFileNodeEdit {
     pub file_id: FileId,
-    pub edits: Vec<AtomEdit>,
+    pub edits: Vec<AtomTextEdit>,
 }
```

Comment:
I think we should do another small refactoring here

* rename `SourceFileNodeEdit` -> `SourceFileEdit` (this was a search/replace accident apparently)
* change `Vec<AtomEdit>` into `TextEdit`: exposing a list of atomic edits is a poor API, exactly because it makes semantics of consecutive edits uncertain.